SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -1148,9 +1149,19 @@ public async Task<IActionResult> YourRecommendations_Post(YourRecommendationsVie
                 {
                     emailApi.SendReferenceNumberEmail(userDataModel.EmailAddress, userDataModel.Reference);
                 }
-                catch (NotifyClientException)
+                catch (EmailSenderException e)
                 {
-                    ModelState.AddModelError(nameof(viewModel.EmailAddress), "Enter a valid email address");
+                    switch (e.Type)
+                    {
+                        case EmailSenderExceptionType.InvalidEmailAddress:
+                            ModelState.AddModelError(nameof(viewModel.EmailAddress), "Enter a valid email address");
+                            break;
+                        case EmailSenderExceptionType.Other:
+                            ModelState.AddModelError(nameof(viewModel.EmailAddress), "Unable to send email due to unexpected error. Please uncheck this box and make a note of your reference number before you continue.");

```

Comment:
We've discussed this, and the error message handling will change a little bit to be more user friendly

Reply:
Preferably the error should be in only one paragraph, so there's no overhead in doing this.